specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,446 @@
+          "NotebookWorkspacesResource"
+        ],
+        "operationId": "NotebookWorkspaces_Start",
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceStart": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceStart.json"
+          }
+        },
+        "description": "Starts the notebook workspace",
+        "parameters": [
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted. The start operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NotebookWorkspaceCreateParameters": {
+      "description": "Parameters to create a notebook workspace resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ]
+    },
+    "NotebookWorkspaceListResult": {
+      "description": "A list of notebook workspace resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Array of notebook workspace resources",
+          "items": {
+            "$ref": "#/definitions/NotebookWorkspace"
+          }
+        }
+      }
+    },
+    "NotebookWorkspace": {
+      "description": "A notebook workspace resource",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NotebookWorkspaceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ]
+    },
+    "NotebookWorkspaceProperties": {
+      "description": "Properties of a notebook workspace resource.",
+      "type": "object",
+      "properties": {
+        "notebookServerEndpoint": {
+          "description": "Specifies the endpoint of Notebook server.",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
```

Comment:
>status [](start = 9, length = 6)

this should be provisioningState and have valid terminal states.  This is required for your PUT operation to be handled async correctly.  [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put)

Reply:
Hi @pilor , my understanding is that provisioningState is only required for tracked resources and resources following 201 + provisioningState pattern. 
These APIs here are proxy APIs. We also use 202 + Location header async pattern for these APIs, which based on our testing are working correctly. Since most of our other proxy resources already work this way, can we continue to following the same pattern?

Reply:
The async pattern is the same for both proxy and tracked resources.  I don't see any other instance of using a "status" property to represent the provisioning status of a resource in cosmos DB swaggers.  If a status property is needed you should be using the normal 201 + provisioningState pattern.  If not, 202 + location header is "ok" but not the preferred approach

Reply:
@shurd Kindly ping

Reply:
We are working on making this change to our RP. I will update once we have resolved it.

Reply:
@shurd If I understand correctly, there will be another PR to change it. right? Is this one resolved? @pilor